runtime/appruntime/config/config.go

```diff
@@ -262,3 +263,13 @@ type RedisDatabase struct {
 	// without having to coordinate and persist database index ids.
 	KeyPrefix string `json:"key_prefix"`
 }
+
+type MetricsExporterType string
+
+const (
+	MetricsExporterTypeLogsBased MetricsExporterType = "logs-based"
```

Comment:
Nit: Can we make this `logs_based` maybe? Or was this format inspired by something else?

Reply:
I've seen other constants having `-` instead of `_`, but I don't think all constants have `-`. I prefer `_` too!